manager/controllers/utils.go

```diff
@@ -47,6 +47,17 @@ func LogAndRespNotFound(c *gin.Context, err error, respMsg string) {
 	c.AbortWithStatusJSON(http.StatusNotFound, utils.ErrorResponse{Error: respMsg})
 }
 
+func createOrderFields(fieldExprs database.AttrMap, order string, enteredField string) string {
+	var orderFields string
+	if enteredField == "os_name" {
```

Comment:
Actually do we need to create this special rule for `os_name`? Couldn't we include all three columns (os_name, os_major, os_minor) into the sort fields? Then we could use OS sort on query params layer (`sort=os_name&sort=os_major&sort=os_minor`).

Reply:
On the master branch where the gorm labels for the respective columns are `osname`, `osmajor`, `osminor` currently, I have tried `api/patch/v1/systems/?sort=osname&sort=osmajor&sort=osminor`, but it resulted in an incorrect result.

Reply:
Ah, I've checked that. You need to use different notation: `?sort=osname,osmajor,osminor`. Then it should works. It's due to: https://github.com/RedHatInsights/patchman-engine/blob/c8ea096df5aeea795f918556af429b90ae840f8c/manager/controllers/utils.go#L54
@semtexzv please, why don't we use `sort=os_name&sort=os_major&sort=os_minor` instead?

Reply:
@Josca There are 2 things that we need to consider:
1. From the UI side, we can send only one column name. In order to send `sort=os_name&sort=os_major&sort=os_minor`, we need a hackish code.
2. Url params may be prone to regressions. Later, if we forget the order of columns os_name -> os_major -> os_minor, it will result in an incorrect result